### DISCUSSION TITLE
WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version

### DISCUSSION THREAD

**Discussion by jttttttttttttt** [PyMySQL/mysqlclient#648] [Q&A] [answered]
Recently, when the program restarts, it will issue: 
        WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version.

I install according to the requirements on the official website:
        $ sudo apt-get install python3-dev default-libmysqlclient-dev build-essential # Debian / Ubuntu

I have found many solutions, one of which is rolled out libmysqlclient21=8.0.28-0ubuntu4.
What should I do.

**Comment by methane** [comment, marked as answer]
do you use `db.ping(True)` or `db.ping(False`?

**Comment by jttttttttttttt** [comment]
I think this problem will be solved soon.        https://bugs.mysql.com/bug.php?id=112089
